VISION/03-Connectors/User-Experience.md

```
@@ -6,26 +6,36 @@ A Connector is a bridge between the agent and an external service (email, calend
+Added connector 'gmail' (gmail / imap, status setup, auth authenticated).
+Next: run `mimir connector resume gmail` to activate it, then `mimir connector sync gmail` to sync.
 
-$ agent connector add homeassistant
-Connector: Home Assistant
-URL: http://homeassistant.local:8123
-Long-lived access token: ████████
-Connected! Found 47 entities (lights, sensors, cameras).
+$ mimir connector add photos --backend local watch_dir=/home/me/Pictures
+Added connector 'photos' (photos / local, status setup, auth unauthenticated).
+Next: run `mimir connector resume photos` to activate it, then `mimir connector sync photos` to sync.
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)